src/psyclone/tests/psyir/frontend/fparser2_generate_psyir_test.py

```diff
@@ -43,7 +43,7 @@ class Fparser2Reader. This method translates an fparser2 parse tree to
 
 from fparser.common.readfortran import FortranStringReader
 from psyclone.errors import GenerationError
-from psyclone.psyir.nodes import Container, Routine, CodeBlock
+from psyclone.psyir.nodes import Container, Routine, CodeBlock, FileContainer
```

Comment:
pylint says CodeBlock is not used in this file

Reply:
Hmm, there is a theme here.